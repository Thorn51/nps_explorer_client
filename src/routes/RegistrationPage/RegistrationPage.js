import React from "react";
import Navigation from "../../components/Navigation/Navigation";
import Hero from "../../components/Hero/Hero";
import "./RegistrationPage.css";

export default function RegistrationPage() {
  return (
    <>
      <Navigation />
      <Hero title="National Parks Explorer" />
    </>
  );
}
