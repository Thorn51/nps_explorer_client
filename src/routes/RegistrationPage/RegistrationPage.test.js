import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import RegistrationPage from "./RegistrationPage";

describe("<RegistrationPage />", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<RegistrationPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("Renders the UI as expected", () => {
    const tree = renderer.create(<RegistrationPage />).toJSON();
    expect("tree").toMatchSnapshot();
  });
});
