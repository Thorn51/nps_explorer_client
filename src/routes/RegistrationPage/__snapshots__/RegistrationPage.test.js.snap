// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegistrationPage /> Renders the UI as expected 1`] = `"tree"`;
